.changes/unreleased/tinymce-TINY-10820-2024-05-21.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: 'Focusing on `contenteditable="true"` element when using `editabe_root: false`
+  causing selection to be shifted.'
```

```suggestion
body: 'Focusing on `contenteditable="true"` element when using `editable_root: false`
  causing selection to be shifted.'
```

This is a bit wide this only affects inline mode since if I change the fiddle to iframe mode it works fine since it doesn't need the bookmark stuff since each window has it's own selection state.

---

Added some more details 👍